include/vapor/SliceGridAlongPlane.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include <vapor/glutil.h>
+#ifdef Darwin
+    #include <OpenGL/gl.h>
+#else
+    #include <GL/gl.h>
+#endif
+
+#include <glm/glm.hpp>
+
+class RegularGrid;
+
+struct planeDescription {
+    std::vector<double> origin;
+    std::vector<double> rotation;
+    std::vector<double> boxMin;
+    std::vector<double> boxMax;
+};
+
+glm::tvec3<double, glm::highp> getOrthogonal(
+    const glm::tvec3<double, glm::highp>& u
+);
+
+void rotate(
```

Comment:
These probably should not be in the global namespace.

Reply:
Agreed. @sgpearse you don't need to include these definitions in the public header file if they are only used internally by SliceGridAlongPlane.cpp

Reply:
Fixed.